### ISSUE TITLE
Self signed certificate failing

### ISSUE THREAD

**Issue by aramacciotti** [swellaby/vsts-mirror-git-repository#8]
I'm getting the below error using the new setting:

```
2018-05-07T06:22:05.6895776Z [command]C:\agents\2.133.3\externals\git\cmd\git.exe "-c http.sslVerify=false" clone --mirror https://...........
2018-05-07T06:22:05.7176930Z Unknown option: -c http.sslVerify=false
2018-05-07T06:22:05.7178931Z usage: git [--version] [--help] [-C <path>] [-c name=value]
2018-05-07T06:22:05.7179372Z            [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
2018-05-07T06:22:05.7179623Z            [-p | --paginate | --no-pager] [--no-replace-objects] [--bare]
2018-05-07T06:22:05.7179837Z            [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
2018-05-07T06:22:05.7180004Z            <command> [<args>]
```
Possibly it's just a matter of the quotes?
The other question is: such a setting is used also at pushing to the target repository, right?

**Comment by traviskosarek** [comment]
It looks like the format I used when executing the command interpreted the commands as a string on your system. Solution was to simply separate the two into a string array.

Instead of "-c http.sslVerify=false" it became ["-c", "http.sslVerify=false"].

My mistake :(

I am merging a new PR shortly that should resolve this issue. I also modified the checkboxes so that there is an option for both the Source Repository and the Destination Repository.

For your case, I think you want to uncheck both checkboxes.
